### DISCUSSION TITLE
Organization project board do not show labels.

### DISCUSSION THREAD

**Discussion by GeorgeHop** [community/community#33993] [Other Feature Feedback, Questions, & Ideas]
When I move my project to organization all labels in tasks appear but in the new project board view label isn't show.

Old project on my github profile
![image](https://user-images.githubusercontent.com/47904385/192054434-472a6a45-ffb0-49bd-8080-a42ea5464275.png)

New project moved to organization
![image](https://user-images.githubusercontent.com/47904385/192054549-0d678c5e-2c16-466b-ae7a-7bc57b7a5d33.png)

How to show labels in organization task board?

**Comment by 0xblank** [comment]
Any update on this topic ?
It's really annoying to not be able to view the labels

**Comment by savin000** [comment]
Press on the arrow on your view (1st attachment). In opened popup under "Configuration" select "fields". Here you can customise your hidden fields (2nd attachment). Labels are hidden by default I guess.

<img width="413" alt="image" src="https://user-images.githubusercontent.com/58307835/225430525-b6c232c1-acc1-4ff7-b4d5-cc7456b9eac4.png">

<img width="209" alt="image" src="https://user-images.githubusercontent.com/58307835/225430973-869cc8cd-7ca8-4d1e-aa41-ee17eb0d670a.png">

**Reply by michaelviveiros** [reply]
Great, thanks!
